app/javascript/components/search/results/ResultsPanel.js

```diff
@@ -15,38 +14,50 @@ const ResultsPanel = props => {
   const results = searchContext.results
   let panelContent
   if (searchContext.isError) {
-    panelContent =
-      <div className="error-panel">
-        <FontAwesomeIcon icon={faExclamationCircle}/>
-        Sorry, an error has occurred.  Support has been notified.
-        Please try again.
+    panelContent = (
+      <div className="error-panel  col-md-6 col-md-offset-3">
+        <FontAwesomeIcon
+          icon={faExclamationCircle}
+          style={{ 'margin-right': '5px' }}
+        />
+        Sorry, an error has occurred. Support has been notified. Please try
+        again. If this error persists, or you require assistance, please contact
+        support at{' '}
```

Comment:
if you just need an extra space to render, use `&nbsp;` or `&#32;` depending on whether you want line breaks to be a possibility on that space